src/commonMain/kotlin/exchange.dydx.abacus/output/Market.kt

```diff
@@ -289,6 +289,8 @@ data class MarketPerpetual(
     val nextFundingAtMilliseconds: Double? = null,
     val openInterest: Double,
     val openInterestUSDC: Double,
+    val openInterestLowerCap: Double? = null,
+    val openInterestUpperCap: Double? = null,
```

Comment:
Just wanna double check - are these values actually nullable in the API response?

Reply:
unfortunately, [yes](https://docs.dydx.exchange/developers/indexer/indexer_websocket), AFAIK